Makefile

```diff
@@ -295,12 +295,6 @@ tools/bin/unconvert: tools/check/go.mod
 tools/bin/failpoint-ctl: go.mod
 	$(GO) build -o $@ github.com/pingcap/failpoint/failpoint-ctl
 
-tools/bin/misspell:tools/check/go.mod
```

Comment:
Why is it useless?
https://github.com/pingcap/tidb/pull/16834/files#diff-b67911656ef5d18c4ae36cb6741b7965L103

Reply:
Because golangci-lint already already has these plugins internally.(such as misspell, ineffassign...)